src/app/clarity/descriptions/loadDescriptions.ts

```diff
@@ -7,8 +7,9 @@ import * as actions from '../actions';
 import { ClarityDescription, ClarityVersions } from './descriptionInterface';
 
 const urls = {
-  descriptions: 'https://ice-mourne.github.io/database-clarity/descriptions.json',
-  version: 'https://ice-mourne.github.io/database-clarity/versions.json',
+  // TODO: change links after merging with main branch
+  descriptions: 'https://database-clarity.github.io/Live-Clarity-Database/descriptions/dim.json',
+  version: 'https://database-clarity.github.io/Live-Clarity-Database/versions.json',
```

Comment:
A little unsure about this. Is this comment outdated or is the intention that we'll merge this PR, and then update these URLs again in a subsequent PR?

Reply:
It was a reminder for me and it is outdated

Reply:
Cool, happy to drop the comment?
```suggestion
  descriptions: 'https://database-clarity.github.io/Live-Clarity-Database/descriptions/dim.json',
  version: 'https://database-clarity.github.io/Live-Clarity-Database/versions.json',
```